styles/components/Event.module.scss

```diff
@@ -0,0 +1,42 @@
+.container {
+  @apply block max-w-full max-h-full w-full aspect-event;
+  @apply m-auto rounded-xl overflow-hidden shadow-l1;
+  @apply relative cursor-pointer;
+  transition: all 0.3s cubic-bezier(0.25, 0.8, 0.25, 1);
```

Comment:
nit but if we want to use this elsewhere might be useful to put it in the tailwind config and use it like `@apply transition-all duration-300 easeOutQuint` or whatever you call it

Reply:
I think we can come back and do that if we see the need.

Reply:
ok, sounds good